src/IntelliTect.Coalesce/DataAnnotations/HttpMethodAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IntelliTect.Coalesce.DataAnnotations
+{
+    /// <summary>
+    /// Allows specifying if this method should use an HTTP method type other than POST.
+    /// </summary>
+    [System.AttributeUsage(System.AttributeTargets.Method)]
+    public class HttpMethodAttribute : System.Attribute
```

Comment:
Requesting documentation, either in https://github.com/IntelliTect/Coalesce/tree/2.0/docs/pages/modeling/model-components/attributes or https://github.com/IntelliTect/Coalesce/blob/2.0/docs/pages/modeling/model-components/methods.rst (or preferably, both locations)

Reply:
(forgot to comment on my review - I think Kelly's and Kevin's suggestions are good to make as well)